server/src/main/resources/application.conf

```diff
@@ -128,6 +130,8 @@ dq {
   s3 {
     bucket = ${?DQ_S3_BUCKET}
   }
+  raw_zip_files_path = ""
```

Comment:
I don't think we need to specify this empty version of the environment variable is accessed using the option syntax.

Reply:
good point